charts/victoria-metrics-cluster/templates/vminsert-deployment.yaml

```
@@ -150,8 +150,10 @@ spec:
       affinity:
 {{ toYaml .Values.vminsert.affinity | indent 8 }}
     {{- end }}
-{{- end }}
+    {{- if .Values.vminsert.extraVolumes }}
       volumes:
         {{- with .Values.vminsert.extraVolumes }}
           {{- toYaml . | nindent 8 }}
         {{- end }}
+        {{- end }}
+{{- end }}
```

i recommend to use like that
```suggestion
      {{- with .Values.vminsert.extraVolumes }}
      volumes:
          {{- toYaml . | nindent 8 }}
      {{- end }}
{{- end }}
```
in this case less logic and `with` is checking if var isn't empty and also fixed end of manifest [PR](https://github.com/VictoriaMetrics/helm-charts/pull/356)

---

This also works, @tenmozes shall I make this change?